hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/ha/MockSCMHAManager.java

```diff
@@ -204,23 +180,27 @@ private Message process(final SCMRatisRequest request)
     }
 
     @Override
-    public RaftServer getServer() {
-      return null;
+    public void stop() {
     }
 
     @Override
-    public RaftGroupId getRaftGroupId() {
+    public RaftServer.Division getDivision() {
       return null;
```

Comment:
Do we allow Division to be null?

Reply:
No, we don't allow null Divison in real env.